webanno-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/rendering/model/VSpan.java

```diff
@@ -40,20 +41,20 @@
     public VSpan(AnnotationLayer aLayer, AnnotationFS aFS, String aType, VRange aOffsets,
             Map<String, String> aFeatures)
     {
-        this(aLayer, new VID(getAddr(aFS)), aType, asList(aOffsets), aFeatures);
+        this(aLayer, new VID(getAddr(aFS)), aType, Collections.singletonList(aOffsets), aFeatures);
```

Comment:
_Arrays.asList_ with just a single argument wastes memory and is less performant than _Collections.singletonList_ because singletonList is not backed by an array and just has a reference to that one item. However, its may not worth it if you say the readability suffers disproportional. Should I revert it?